scipp-imaging-2D-Bench.ipynb

```diff
@@ -0,0 +1,1585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to start\n",
+    "\n",
+    "Before starting you must ensure that `scipp`, `mantid`, and `wfm_stitching` are on your `PYTHONPATH`.\n",
+    "\n",
+    "For the first two follow instructions at: https://scipp.readthedocs.io/en/latest/getting-started/installation.html.\n",
+    "The last one can be acquired from https://git.esss.dk/wedel/wfm_stitching\n",
+    "\n",
+    "Additionally, to other package dependencies are `fabio` and `astropy`: `conda install fabio astropy`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# created by Owen Arnold (optional e-mail) on 2018-09-19\n",
+    "# last modified by Peter M. Kadletz (peter.kadletz@esss.se) on 2018-11-12\n",
+    "import fabio\n",
+    "import numpy as np\n",
+    "import glob\n",
+    "import sys\n",
+    "import csv\n",
+    "import os\n",
+    "\n",
+    "import scipp as sc\n",
+    "import numpy as np\n",
+    "from scipp import Dim\n",
+    "\n",
+    "from mantid.simpleapi import *\n",
+    "from mantid.api import AlgorithmManager\n",
+    "\n",
+    "from wfm_stitching import WFMProcessor\n",
+    "\n",
+    "# defining grouping of 2D detector pixels\n",
+    "grouping_number = 12\n",
+    "nx_target = grouping_number\n",
+    "ny_target = grouping_number\n",
+    "\n",
+    "# Number of acquired pulses per measurement\n",
+    "pulse_nr_reference = 1.0 / 770956\n",
+    "pulse_nr_sample = 1.0 / 1280381\n",
+    "pulse_nr_sample_elastic = 1.0 / 2416839\n",
+    "pulse_nr_plastic = 1.0 / 2614343\n",
+    "\n",
+    "# let's get the process started:\n",
+    "experiment_dir = \"/home/dtasev/dev/scipp_imaging/RAL_Mantid_Sep2018/\"\n",
+    "\n",
+    "tofs_path = os.path.join(experiment_dir, 'data_GP2', 'metadata', 'GP2_Stress_time_values.txt')\n",
+    "sample_path = os.path.join(experiment_dir, 'data_GP2', 'Stress Experiments', '2) R825')\n",
+    "sample_elastic_path = os.path.join(experiment_dir, 'data_GP2', 'Stress Experiments', '3) R825 600 Mpa')\n",
+    "ob_path = os.path.join(experiment_dir, 'data_GP2', 'Stress Experiments', '1) R825 Open Beam')\n",
+    "\n",
+    "# These use the range notation of [start, stop, step]\n",
+    "# for R825\n",
+    "frame_parameters = [(15450,22942,64),\n",
```

Comment:
Could you add more information here. This is more public facing that the scripts you originally worked from. What are these values, what units. Maybe could immediately be encoded as scipp variables as a self-documenting approach?